Clinical-Genomics/scout
# Add cancer tiers

This PR adds a functionality - cancer tier classification. See #548.

**How to test**:
1. with a cancer case, pick a variant and assign a tier.
1. make sure the tier can be seen on the variants page and general report
1. de-assign the tier and make sure it disappears.

**Expected outcome**:
The functionality should be working

<img width="719" alt="Screenshot 2019-11-23 at 12 20 28" src="https://user-images.githubusercontent.com/758570/69477868-bf871c80-0deb-11ea-9ab3-62610f981c60.png">

**Review:**
